CoreHelp/src/au/gov/asd/tac/constellation/help/ConstellationHelpDisplayer.java

```diff
@@ -237,8 +235,8 @@ public boolean display(final HelpCtx helpCtx) {
         final String helpId = helpCtx.getHelpID();
         LOGGER.log(Level.INFO, "display help for: {0}", helpId);
 
-        final String helpDefaultPath = sep + "constellation" + sep + "CoreFunctionality" + sep + "src" + sep + "au" + sep + "gov"
-                + sep + "asd" + sep + "tac" + sep + "constellation" + sep + "functionality" + sep + "docs" + sep + "about-constellation.md";
+        final String helpDefaultPath = "ext" + sep + "docs" + sep + "CoreFunctionality" + sep + "src" + sep + "au" + sep + "gov"
```

Comment:
This needs to start with a sep before the "ext" for online help

Reply:
Ok thanks. We are only fixing for Off line help at the moment.